addons/l10n_eu_service/wizard/wizard.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="eu_service_view" model="ir.ui.view">
+            <field name="name">l10n_eu_service.wizard.form</field>
+            <field name="model">l10n_eu_service.wizard</field>
+            <field name="arch" type="xml">
+                <form string='EU service'>
+                    <p>
+                        Based on the options selected below, this wizard will create one
+                        fiscal position mapping for each EU country to which you are selling
+                        services.
+                        Each fiscal position will automatically map your national VAT tax for
+                        services to the corresponding VAT tax in the country your customer
+                        belongs to.
+                        <br/>
+                        You can use the wizard again later to add more countries.
+                    </p>
```

Comment:
@xmo-odoo It was initially a set a 3 paragraphs, but it was rather ugly due to the default `<p>` margins. If you have a better idea without requiring extra CSS... :-)